### ISSUE TITLE
Interactive element does not meet minimum size nor spacing - accordions

### ISSUE THREAD

**Issue by GaryRidgway** [uiowa/uiowa#9302]
# Problem

It seems that for some reason siteimprove is flagging the _Second_ Link in the _First_ accordion item on pages as having the wrong interactive size.

Here is a few examples of this:

- [Example one](https://my2.siteimprove.com/Inspector/1005109/A11Y/Page?pageId=65055911727&impmd=6F4B1BAAA780A4D7A3CA00D93433A366&issue=cantTell+failed&wcag=twopointtwo&decision=dismissed+falsePositive+none#/sia-r113/failed/OXdYd1JHbXU=;ZDIyYzk4ZGY=)
- [Example two](https://my2.siteimprove.com/Inspector/1005109/A11Y/Page?pageId=65059236606&impmd=6F4B1BAAA780A4D7A3CA00D93433A366&issue=cantTell+failed&wcag=twopointtwo&decision=dismissed+falsePositive+none#/sia-r113/failed/Wjhxd0x3Z0U=;MjQ5ODljMTE=)

# Proposed solution
Understand why that specific link is flagged, and then find a way to resolve it.

**Comment by bspeare** [comment]
Ticket response:

> Thanks for reaching out!
> 
> Unfortunately, our support team isn't able to assist at a consultant level with Accessibility issues due to legal requirements about certifications for Web Content Accessibility Guidelines. I would recommend first looking at the ALFA and WCAG documentation provided in the page report for a given issue. It will show up on the left-hand side of the screen under the "Why is this important?" dropdown. If the content there doesn't answer your questions, then you'll need to speak with our Accessibility team regarding these issues.
> 
> To have this reviewed by our Accessibility team, you'll want to go into the page report for one of these issues and use the gear icon within the issue to mark the issue as a "False positive". You can also use a CSS selector to report multiple false positives at one time. This will generate a report that is sent to our accessibility product team for review. Once they've reviewed it, they will reach out to you to confirm the issue and award you the points you would have earned for fixing it. You can review this report's progress from the [Accessibility > Activity page in the platform.](https://my2.siteimprove.com/Accessibility/NextGen/Activity)
> 
> You can learn more about this from our [Next Generation accessibility: How to dismiss occurrences of accessibility issues](https://help.siteimprove.com/support/solutions/articles/80000962686-accessibility-how-to-dismiss-occurrences-of-accessibility-issues) help article. 
> 
> You can also speak with your Customer Success Executive Erika (ebor@siteimprove.com) regarding what options you have available for professional services like Accessibility consultations if the issue you're looking into isn't necessarily a false positive.
> 
> Let me know if you have any questions!
> 
> 
> Thank you,
> --
> Cynthia O.
> Customer Support Engineer
> Siteimprove, Inc.
> --
> Ticket ID: 4350789
> Ticket URL: https://help.siteimprove.com/helpdesk/tickets/4350789

**Comment by joewhitsitt** [comment]
Welp. Here we go. https://my2.siteimprove.com/Accessibility/1005109/NextGen/Activity/1/Updated/Desc?tagId=2&pageSize=20&conformance=0,1,3,4&pageSegments=&siteTarget.wcagVersion=22&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2&siteTargetIssueKinds=1,2&wcagVersion=22

<img width="1356" height="692" alt="Image" src="https://github.com/user-attachments/assets/352c5ced-8a9c-40da-8764-738d1ca0e388" />
<img width="594" height="273" alt="Image" src="https://github.com/user-attachments/assets/c9978422-9e53-49cf-af2b-e371ef591c81" />

This appears to be a site by site specific reporting tool though so unless their scanner is updated to account for this, we might have to take additional measures to "hide" this content even though we are using a semantic element.

**Comment by joewhitsitt** [comment]
what grinds my gears is that this is not being reported by the browser extension so there isn't a great way to test this, but this combination could "trick" the scanner if SiteImprove doesn't help with a platform wide solution.

```css
// Hack for SiteImprove scans flagging links' target size in closed accordion items.
// Content still available to browser find tools.
details:not([open]) .accordion__content a {
  display: block;
  pointer-events: none;
  width: 24px;
  height: 24px;
}
```

**Comment by joewhitsitt** [comment]
Still awaiting review https://github.com/uiowa/uiowa/issues/9302#issuecomment-3503451856 (5 pages now). Going to unassign myself for now but I will continue to check in on it or pass along any emails I get related to this.

**Comment by joewhitsitt** [comment]
I am curious if someone can search for this platform wide? I _just_ got a review back. They "rejected" it but because they couldn't see that it was still an issue? This does not show as an issue for Facilities anymore.

>I'm rejecting this because I can't see the flagged occurrence in the Siteimprove platform any more. So I will assume that this isn't a problem any more. If it is, please submit another false positive report.

**Comment by GaryRidgway** [comment]
I completely agree with Brian on the draft generally, but I think you have also made it clear even from the title that this is about accordions. You could probably just add a note at the bottom that says something like `NOTE: This false positive applies _ONLY_ to accordions, if you are seeing it elsewhere, blah blah blah`

Additionally, it might be nice to have a screenshot of a siteimprove screen where this is happening as a secondary visual link for a user? if they see a highlighted area under an accordion that is closed, it might be a good trigger for them to know that they are in the right place.
